test/nightly/azureResourceMongoDB.test.ts

```diff
@@ -7,34 +7,74 @@ import * as assert from 'assert';
 import { CosmosDBManagementModels } from 'azure-arm-cosmosdb';
 import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
 import * as vscode from 'vscode';
+import * as request from 'request-promise';
 import { randomUtils } from '../../extension.bundle';
 import { longRunningTestsEnabled, testUserInput } from '../global.test';
-import { resourceGroupsToDelete, client } from './global.resource.test';
+import { WebResource, ServiceClientCredentials } from "ms-rest";
+import { resourceGroupsToDelete, client, testAccount } from './global.resource.test';
 
 suite('MongoDB action', async function (this: ISuiteCallbackContext): Promise<void> {
     this.timeout(20 * 60 * 1000);
+    let resourceGroupName: string;
+    let accountName: string;
 
     suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
         if (!longRunningTestsEnabled) {
             this.skip();
         }
         this.timeout(2 * 60 * 1000);
-    });
-
-    test('create Cosmos DB account', async () => {
-        const resourceGroupName: string = randomUtils.getRandomHexString(12);
+        resourceGroupName = randomUtils.getRandomHexString(12);
         // Cosmos DB account must have lower case name
-        const accountName: string = randomUtils.getRandomHexString(12).toLowerCase();
+        accountName = randomUtils.getRandomHexString(12).toLowerCase();
         resourceGroupsToDelete.push(resourceGroupName);
+    });
+
+    test('Create MongoDB account', async () => {
         const testInputs: string[] = [accountName, 'MongoDB', '$(plus) Create new resource group', resourceGroupName, 'West US'];
         await testUserInput.runWithInputs(testInputs, async () => {
             await vscode.commands.executeCommand('cosmosDB.createAccount');
         });
         const createAccount: CosmosDBManagementModels.DatabaseAccount | undefined = await client.databaseAccounts.get(resourceGroupName, accountName);
         assert.ok(createAccount);
     });
+
+    test('Create MongoDB Database', async () => {
+        const databaseName: string = randomUtils.getRandomHexString(12);
+        const collectionName: string = randomUtils.getRandomHexString(12);
+        const testInputs: string[] = [testAccount.getSubscriptionContext().subscriptionDisplayName, `${accountName} (MongoDB)`, databaseName, collectionName];
+        await testUserInput.runWithInputs(testInputs, async () => {
+            await vscode.commands.executeCommand('cosmosDB.createMongoDatabase');
+        });
+        const response: string = await getMongoDBDatabase(await getUrl(databaseName));
+        assert.equal(JSON.parse(response).properties.id, databaseName);
+    });
+
+    async function getUrl(Path: string): Promise<string> {
```

Comment:
This seems kind of complicated and prone to errors.  I think an easier way to verify that the mongo DB got created would be to list all the databases and find the one that was just created.

![image](https://user-images.githubusercontent.com/5290572/67910100-b41f3900-fb3e-11e9-9484-97eb845aa748.png)

The code can be found here: https://github.com/microsoft/vscode-cosmosdb/blob/master/src/mongo/tree/MongoAccountTreeItem.ts#L71-L72